pytest.ini

```diff
@@ -3,7 +3,7 @@ env =
     DEBUG=1
     TEST=1
 DJANGO_SETTINGS_MODULE = posthog.settings
-addopts = -p no:warnings --reuse-db --ignore=posthog/user_scripts
+addopts = -p no:warnings --reuse-db --ignore=posthog/user_scripts --ignore=posthog/clickhouse/migrations/0097_session_replay_events_v2_test.py
```

Comment:
@Gilbert09 do you remember the context for this?
(am thinking of unwinding the existence of this table)